filters/style_manager.py

```diff
@@ -114,7 +114,14 @@ def responseComplete(self):
             style_manager = qgis_layer.styleManager()
             if '' in style_manager.styles():
                 style_manager.renameStyle('', 'default')
-            project.write()
+                project.write()
+                # Make sure properties are saved.
+                # So GetCapabilities are working.
+                project.clear()
```

Comment:
Need to add this line, so GetCapabilities work correctly and returns the new style lists.